.taskcluster.yml

```diff
@@ -80,198 +80,214 @@ tasks:
                       else:
                           $if: 'tasks_for == "cron"'
                           then: '${ownTaskId}'
+
+                  pullRequestAction:
+                      $if: 'tasks_for == "github-pull-request"'
+                      then: ${event.action}
+                      else: 'UNDEFINED'
+
+                  releaseAction:
+                      $if: 'tasks_for == "github-release"'
+                      then: ${event.action}
+                      else: 'UNDEFINED'
               in:
-                  $let:
-                      level:
-                          $if: 'tasks_for in ["github-push", "github-release", "action", "cron"] && repoUrl == "https://github.com/mozilla-mobile/reference-browser"'
-                          then: '3'
-                          else: '1'
-                  in:
-                      taskId: '${ownTaskId}'
-                      taskGroupId:
-                          $if: 'tasks_for == "action"'
-                          then:
-                              '${action.taskGroupId}'
-                          else:
-                              '${ownTaskId}'  # same as taskId; this is how automation identifies a decision task
-                      schedulerId: '${trustDomain}-level-${level}'
-                      created: {$fromNow: ''}
-                      deadline: {$fromNow: '1 day'}
-                      expires: {$fromNow: '1 year 1 second'}  # 1 second so artifacts expire first, despite rounding errors
-                      metadata:
-                          $merge:
-                              - owner: "${ownerEmail}"
-                                source: '${repoUrl}/raw/${head_sha}/.taskcluster.yml'
-                              - $if: 'tasks_for in ["github-push", "github-pull-request"]'
-                                then:
-                                    name: "Decision Task"
-                                    description: 'The task that creates all of the other tasks in the task graph'
-                                else:
-                                    $if: 'tasks_for == "action"'
+                  $if: >
+                    tasks_for in ["action", "cron"]
+                    || (tasks_for == "github-pull-request" && pullRequestAction in ["opened", "reopened", "synchronize"])
+                    || (tasks_for == "github-push" && head_branch[:10] != "refs/tags/" && head_branch != "staging.tmp" && head_branch != "trying.tmp")
+                    || (tasks_for == "github-release" && releaseAction == "published")
+                  then:
```

Comment:
The diff is convoluted, but past this point, I just shifted everything right by 4 spaces.